$NetBSD: patch-aa,v 1.1.1.1 2010/10/19 16:37:40 abs Exp $
# Add support for NetBSD - copy of FreeBSD except for uname -m amd64 vs x86_64

--- configure.orig	2010-09-14 06:00:09.000000000 +0000
+++ configure
@@ -97,6 +97,14 @@ build_config()
 		echo "RDYNAMIC=-rdynamic"
 		echo "LDFLAGS+=-Wl,--no-undefined -Wl,--as-needed"
 		;;
+		NetBSD)
+		[ "$(uname -m)" = "amd64" ] && echo "ARCH_X86_64=1" && echo "CFLAGS+=-DARCH_X86_64"
+		echo "OPTIMISATIONS+=-ffast-math"
+		echo "CFLAGS+=-pthread"
+		echo "SHFLAGS=-shared"
+		echo "RDYNAMIC=-rdynamic"
+		echo "LDFLAGS+=-Wl,--no-undefined -Wl,--as-needed"
+		;;
 		*)
 		;;
 		esac
@@ -171,7 +179,7 @@ case $targetos in
 	Darwin)
 	LIBSUF=".dylib"
 	;;
-	Linux|FreeBSD)
+	Linux|FreeBSD|NetBSD)
 	LIBSUF=".so"
 	;;
 	*)
